### PR TITLE
8.2: New version 1.4.0

### PR DESCRIPTION
**Pull request by Wescoeur**
- Try to open device and start HTTP server before notifying the user
- Install pyc and pyo files

### CODE REVIEW

**Comment by Wescoeur** [comment]
Koji build (scratch, v8.2-incoming): http://koji.xcp-ng.org/taskinfo?taskID=61784

File: SPECS/http-nbd-transfer.spec [hunk 1]
```diff
@@ -23,17 +26,21 @@ Set of tools to transfer NBD requests to a HTTP server.
 
 %build
 make
+PYTHON=%{__python2} %{__python2} ./setup.py build
 
 %install
 %make_install PREFIX=%{_prefix} DESTDIR=$RPM_BUILD_ROOT
+PYTHON=%{__python2} %{__python2} ./setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
```
Comment (stormi) [inline SPECS/http-nbd-transfer.spec]: IIRC `$RPM_BUILD_ROOT` is best defined as `%{buildroot}` nowadays.

**Review by Nambrok — APPROVED** [review]

**Comment by Wescoeur** [comment]
Koji build (scratch, v8.2-incoming): http://koji.xcp-ng.org/taskinfo?taskID=62469

**Review by stormi — APPROVED** [review]

**Comment by Nambrok** [comment]
Koji build (v8.2-incoming): https://koji.xcp-ng.org/taskinfo?taskID=62751

**Comment by stormi** [comment]
Koji build to v8.2-incoming: https://koji.xcp-ng.org/buildinfo?buildID=3436

**Pull request merged**